documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1208,51 +1208,46 @@ The `connect.decimal.precision` schema parameter contains an integer representin
 |===
 
 [[sqlserver-deploying-a-connector]]
-== Deploying the SQL Server connector
+== Deployment
 
 ifdef::community[]
-If you have already installed https://zookeeper.apache.org[Zookeeper], http://kafka.apache.org/[Kafka], and {link-kafka-docs}.html#connect[Kafka Connect], then using {prodname}'s SQL Server` connector is easy.
-Simply download the https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/{debezium-version}/debezium-connector-sqlserver-{debezium-version}-plugin.tar.gz[connector's plug-in archive], extract the JARs into your Kafka Connect environment, and add the directory with the JARs to {link-kafka-docs}/#connectconfigs[Kafka Connect's `plugin.path`].
-Restart your Kafka Connect process to pick up the new JARs.
+With link:https://zookeeper.apache.org[Zookeeper], http://kafka.apache.org/[Kafka], and {link-kafka-docs}.html#connect[Kafka Connect] installed, the remaining tasks to deploy a {prodname} SQL Server connector are to download the https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/{debezium-version}/debezium-connector-sqlserver-{debezium-version}-plugin.tar.gz[connector's plug-in archive], extract the JAR files into your Kafka Connect environment, and add the directory with the JAR files to {link-kafka-docs}/#connectconfigs[Kafka Connect's `plugin.path`].
+Restart your Kafka Connect process to pick up the new JAR files.
+
+If you need immutable containers, see link:https://hub.docker.com/r/debezium/[{prodname}'s container images] for Zookeeper, Kafka, SQL Server and Kafka Connect with the SQL Server connector already installed and ready to run. You can also link:https://debezium.io/blog/2016/05/31/Debezium-on-Kubernetes/[run {prodname} on Kubernetes and OpenShift].
```

Comment:
So I find the "If you need" sounds a bit odd. Perhaps "If you're working with..."? What's really needed in life, after all :)

Reply:
Sure!  Updated in the recent commits, which reflect your comments, and which I just pushed to this PR.